### PR TITLE
Open local address in default browser

### PR DESCRIPTION
**Pull request by webtaculars**
Open local address in default browser by using option --local.
`serve --local` 

**EDIT**
Command is now `serve --open`

### CODE REVIEW

**Comment by leo** [comment]
👍 I'd rather call it `--open`

**Comment by webtaculars** [comment]
Yeah 😀. It should be `--open'. I will update it.

**Comment by webtaculars** [comment]
@leo I have changed `--local` to `--open`. Need to update anything else?

File: lib/listening.js [hunk 1]
```diff
@@ -67,6 +68,13 @@ module.exports = coroutine(function*(server, current, inUse, clipboard) {
       } catch (err) {}
     }
 
+    if(open) {
```
Comment (leo) [inline lib/listening.js]: Please add a space before the `(`

File: lib/listening.js [hunk 2]
```diff
@@ -67,6 +68,13 @@ module.exports = coroutine(function*(server, current, inUse, clipboard) {
...
+      try {
+        opn(localURL)
+        message += `\n\n${chalk.grey('Local address has opened in browser!')}`
```
Comment (leo) [inline lib/listening.js]: Let's not show any additional messages in the terminal!

File: lib/listening.js (hunk 1 above)
Comment (leo) [inline lib/listening.js]: Also please test for `isTTY`

File: bin/serve.js [hunk 3]
```diff
@@ -108,7 +109,7 @@ detect(port).then(open => {
   server.listen(
     port,
     coroutine(function*() {
-      yield listening(server, current, inUse, flags.noClipboard !== true)
+      yield listening(server, current, inUse, flags.noClipboard !== true, flags.open === true)
```
Comment (leo) [inline bin/serve.js]: Why the comparison? `flags.open` should be enough

**Comment by webtaculars** [comment]
@leo Updated all the changes you asked.

File: bin/serve.js [hunk 4]
```diff
@@ -50,7 +50,7 @@ args
   )
   .option('silent', `Don't log anything to the console`)
   .option('no-clipboard', `Don't copy address to clipboard`, false)
-
+  .option('open', `Open local address in browser`, false)
```
Comment (leo) [inline bin/serve.js]: Use ' instead of ` here, please

Comment (webtaculars) [inline bin/serve.js]: should I change it for silent and no-clipboard also?

Comment (leo) [inline bin/serve.js]: @webtaculars No, because they have a ' in the value.

Comment (webtaculars) [inline bin/serve.js]: we can escape it using backslash

Comment (leo) [inline bin/serve.js]: Yes, but prettier might revert that. Try it!

**Comment by leo** [comment]
Thanks! :v:

**Pull request merged**